dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java

```diff
@@ -96,13 +96,7 @@ private Range getRange(Bounds bounds) {
 
 	private List<Annotation> getSortedAnnotations(Range ranks) {
 		return graph.getAnnotationByRank(ranks).stream()
-				.sorted((a1, a2) -> {
-					if (a1.getStart() < a2.getStart()) {
-						return -1;
-					} else {
-						return 1;
-					}
-				})
+				.sorted((a1, a2) -> Integer.compare(a1.getStart(), a2.getStart()))
```

Comment:
This is not completely the same: `Integer.compare` returns 0 if the two integers are equal, and the lambda expression returns 1 in that case.